package.json

```diff
@@ -19,6 +19,7 @@
     "bitfinex-api-node": "^1.2.1",
     "co-fs": "^1.2.0",
     "commander": "^2.13.0",
+    "crypto-js": "^3.1.9-1",
```

Comment:
can you add this dep to gekko broker instead? Which means inside this package instead: https://github.com/askmike/gekko/blob/develop/exchange/package.json

Else people won't be able to install gekko broker as a standalone package: https://www.npmjs.com/package/gekko-broker